pkg/sql/distsql_running.go

```diff
@@ -1172,6 +1173,7 @@ func (r *DistSQLReceiver) Push(
 	if r.resultWriter.Err() == nil && r.ctx.Err() != nil {
 		r.SetError(r.ctx.Err())
 	}
+	r.stats.networkEgressEstimate += int64(row.Size())
```

Comment:
Is this or the statement below expensive to calculate? If we do something like looping that might be slow for large rows/batches, we might consider only doing it when it's needed. That said, it'd be more complex, so we should only do that if it's a real issue.

Reply:
Yeah, I'll want to hide this all behind some setting so we only do it for tenants that run explain analyze, since overhead can be non-negligible